units/patch_intent.go

```diff
@@ -580,8 +580,9 @@ func (j *patchIntentProcessor) buildCliPatchDoc(ctx context.Context, patchDoc *p
 	commit, err := thirdparty.GetCommitEvent(ctx, githubOauthToken, projectRef.Owner,
 		projectRef.Repo, patchDoc.Githash)
 	if err != nil {
-		return errors.Wrapf(err, "could not find base revision '%s' for project '%s'",
-			patchDoc.Githash, projectRef.Id)
+		grip.Warning(
```

Comment:
I'm concerned about not erroring at all; this really could be an error. I'd like to explore the "ConfigChanged" idea.